app/Nova/DuesTransaction.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Panel;
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\MorphMany;
+
+class DuesTransaction extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = 'App\\DuesTransaction';
+
+    public static $with = ['payment', 'package', 'user'];
+
+    /**
+     * Get the displayble label of the resource.
+     *
+     * @return string
+     */
+    public static function label()
+    {
+        return 'Dues Transactions';
```

Comment:
This shows up as duestransactions in staging

Reply:
Where are you seeing that?

Reply:
It was showing up on user detail pages but I can't reproduce it now

Reply:
Found and fixed